tools/nni_cmd/tests/config_files/test_files/test_json.json

```diff
@@ -0,0 +1 @@
+{"field":"test"}
```

Comment:
missing end of line

Reply:
updated, it it necessary to add additional line in all kinds of files? why?

Reply:
it is not mandatory, but it is git/unix convention.